portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase

```diff
@@ -219,14 +219,67 @@ definition {
 	}
 
 	@description = "Verify that a Fieldset can be added multiple times to Form"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanBeAddedToAFormMultipleTimes {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133463 CanBeAddedToAFormMultipleTimes pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		Form.gotoElementSetsTab();
+
+		FormsAdminNavigator.gotoAddElementSet();
+
+		Form.editName(formName = "Element Set 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		Form.closeSidebar();
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Numeric");
+
+		Form.saveElementSet();
+
+		FormsAdminNavigator.gotoBack();
+
+		Navigator.gotoNavItem(navItem = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		for (var element : list "Element Set 1,Element Set 1") {
+			Form.gotoAddElementSet(elementSetTitle = "${element}");
+		}
+
+		Form.save();
+
+		Alert.viewSuccessMessage();
```

Comment:
@fmarcio, this step is already executed inside the macro `Form.save`, therefore you can remove it. :wink:

Reply:
> @fmarcio, this step is already executed inside the macro `Form.save`, therefore you can remove it.

@natocesarrego fixed! :-)